client/signup/steps/user/index.jsx

```diff
@@ -108,14 +112,24 @@ export class UserStep extends Component {
 	}
 
 	UNSAFE_componentWillMount() {
-		const { oauth2Signup, initialContext } = this.props;
+		const { oauth2Signup, initialContext, flowName } = this.props;
+
 		const clientId = get( initialContext, 'query.oauth2_client_id', null );
 
 		this.setSubHeaderText( this.props );
 
 		if ( oauth2Signup && clientId ) {
 			this.props.fetchOAuth2ClientData( clientId );
 		}
+		if ( flowName === 'onboarding' ) {
+			const experimentCheck = this.state.isDesktop
+				? 'registration_email_only_desktop'
+				: 'registration_email_only_mobile';
```

Comment:
Just. calling out that with this check, tablet resolutions will also get assigned to the mobile test. Maybe that's okay since tablet and mobile have similar touch screen experiences, what do you think? Or should we restrict the test to mobile only and exclude tablet?

Reply:
This is anecdotal, but my browsing behavior is essentially the same on tablets and mobile devices so it seems fine to group them together. I guess the exception would be something like an iPad Pro with a physical keyboard attached, but I doubt that would be a large enough group of people to skew the results.

Reply:
I agree, it's the same anecdotal experience for me too. I'll run a quick test of things again.